src/io/config.cpp

```diff
@@ -240,16 +253,6 @@ void Config::Set(const std::unordered_map<std::string, std::string>& params) {
     save_binary = true;
   }
 
-  if (verbosity == 1) {
```

Comment:
I was surprised to see this being removed, but I *think* it's ok. I couldn't find any cases of `Config.Set()` being called without `Config::Str2Map()` being called, and we have many tests on verbosity in the R and Python package which are passing.

Just wanted to say that out loud, for the benefit of other reviewers.